models/articles.models.js

```diff
@@ -0,0 +1,20 @@
+
+const db = require('../db/connection')
+
+exports.fetchArticles = (article_id) => {
+
+    const regex = /\d+/
+
+    if(!regex.test(article_id)){
+        return Promise.reject({ status: 400, msg: 'bad request'})
+    }
```

Comment:
psql will handle it for you if the article_id is not a number. Let psql handle it rather than custom error handling